testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/management/api/SimpleSubsystemsTestCase.java

```diff
@@ -40,6 +40,7 @@
  */
 @RunWith(Arquillian.class)
 @RunAsClient
+//todo move to wildfly-core
```

Comment:
it only tests "simple" subsystems 
aka subsystems that have no resources or configuration options. so no need to have something like that in full. 
If we really need tests for something like that.